sites/platform/static/files/fetch/examples/relationships/headlesschrome

```diff
@@ -2,8 +2,11 @@
     "service": "headlesschrome",
     "ip": "169.254.91.5",
     "hostname": "gvbo7vktgmou2mplnzt4b54hgi.headlesschrome.service._.eu-3.platformsh.site",
-    "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "headlesschrome.internal",
+    "cluster": "rjify4yjcwxaa-master-7rqtwti",
```

Comment:
same please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.headlesschrome.service._.eu-1.platformsh.site",